docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,284 @@
+
+4. For each name server in *Name Server IP* do:
+
+   1. Send *TXT Query* to the name server and collect the [DNS Response].
+
+   2. Go to the next name server IP address if at least one of the following
+      criteria is met:
+
+      1. There is no DNS response.
+      2. [RCODE Name] of the response is not "NoError".
+      3. The AA flag is not set in the response.
+      4. There is no TXT resource record in the response.
+
+   3. If the name server responds with at least one TXT record, then, for each
+      TXT record:
+
+      1. If the record is an [SPF TXT record], then add the lowercased
+         [concatenation][concatenate] of all strings in the TXT record’s data to
+         the *SPF-Policies* set.
+
+   4. Go to the next name server.
+
+5. Compare the set of *SPF-Policies* retrieved from all name servers. If at
+   least one name server has responded with a set of SPF policies and a second
+   name server has responded with a different set of SPF policies, then output
+   *[Z11_INCONSISTENT_SPF_POLICIES]*.
+
+6. If no name server in the *Name Server IP* set responded with an authoritative
+   "NoError" response, then output *[Z11_UNABLE_TO_CHECK_SPF]* and terminate the
+   test.
+
+7. If the *SPF-Policies* set is empty, then output *[Z11_NO_SPF_FOUND]*
+   and terminate the test.
+
+8. For each string in *SPF-Policies* set, if the string is equal to `v=spf1` or
+   starts with `v=spf1` followed by a space, then add it to the *SPFv1-Policies*
+   set.
+
+9. If the *SPFv1-Policies* set is empty, then for each string in the
+   *SPF-Policies* set, output *[Z11_SPF_WRONG_VERSION]* with the version number
+   extracted from the string and the nameservers that returned the string.
+
+10. If the *SPFv1-Policies* set contains more than one element, then output
+    *[Z11_SPF1_MULTIPLE_RECORDS]* with the list of nameservers that returned
+    more than one SPF policy and terminate the test. The next steps assume
+    there is exactly one string in the *SPFv1-Policies* set ("SPF policy").
+
+11. If the *SPF Policy* does not [pass the syntax check][passing the syntax
+    check] for SPF version 1 records, then output *[Z11_SPF1_SYNTAX_ERROR]* and
+    terminate the test.
+
+12. If the *SPF Policy* contains more than one `redirect` modifier, or more than
+    one `exp` modifier, then output *[Z11_SPF1_MULTIPLE_SAME_MODIFIER]*.
+
+13. If the *SPF Policy* contains exactly one `redirect` modifier anywhere except
+    at the end of the policy text, then output *[Z11_SPF1_REDIRECT_NOT_AT_END]*.
+
+14. If the *SPF Policy* contains both a `redirect` modifier and an `all`
+    mechanism, then output *[Z11_SPF1_REDIRECT_AND_ALL]*.
+
+15. If the *SPF Policy* contains at least one `ptr` mechanism, then output
+    *[Z11_SPF1_HAS_PTR]*.
+
+16. Count the number of terms in the *SPF Policy* that are either ("DNS Lookup
+    count"):
+
+    1. A mechanism of type `a`, `mx`, `ptr`, `exists`;
+
+    2. Or a modifier of type `redirect`.
```

Comment:
Is that the same as:
```
Count the total number of terms in the *SPF Policy* that is either a mechanism of type `a`, `mx`, `ptr`, `exists` or a modifier of type `redirect` ("DNS Lookup count").
```
If so, I think that would be easier to read. Else, please clarify.